### ISSUE TITLE
No module named 'vector_2d'

### ISSUE THREAD

**Issue by kovasap** [Conqu3red/LR-SVG-Converter#1]
Hey I was just trying to use this tool and got the following error:

```
> pip install -r requirements.txt
Collecting lrtools==1.0.2
  Downloading lrtools-1.0.2-py3-none-any.whl (8.7 kB)
Collecting svgelements==1.6.3
  Downloading svgelements-1.6.3-py2.py3-none-any.whl (112 kB)
     |████████████████████████████████| 112 kB 8.4 MB/s 
Installing collected packages: svgelements, lrtools
Successfully installed lrtools-1.0.2 svgelements-1.6.3
> python3 main.py
--- Line Rider SVG Track generator ---
### Created by Conqu3red (Conqu3red#2054 on discord)
### Enter a file name (example: test.svg): basic.svg
### Enter save format
1: TRK file (small file size)
2: JSON (linerider.com)
> 2
### Enter mode
1: Accurate (recommended - default)
2: approximate
> 1
Traceback (most recent call last):
  File "/home/kovas/LR-SVG-Converter/main.py", line 39, in <module>
    import lrtools.trkformat as trkformat
  File "/home/kovas/.local/lib/python3.9/site-packages/lrtools/trkformat.py", line 2, in <module>
    from vector_2d import Vector as Vector2d
ModuleNotFoundError: No module named 'vector_2d'
```

My input file is 
![basic](https://user-images.githubusercontent.com/8763010/155013037-ec11f26d-0bb0-4948-9427-1eb548343601.svg)

My objective is to turn line graphs like the one above, or like this one:

![image](https://user-images.githubusercontent.com/8763010/155013133-aedff91e-e5c9-4643-b470-76d9a0429c12.png)

Into line rider tracks.  If there's a better library to accomplish this I'd be happy to know about it!

**Comment by kovasap** [comment]
Is your [`trk-python`](https://github.com/Conqu3red/trk-python) repo perhaps missing a dependency on [`vector_2d`](https://github.com/Conqu3red/trk-python/blob/9aa59732faf639c84fda69ec899d66e0f99d5fd3/lrtools/jsonformat.py#L1)?

**Comment by kovasap** [comment]
Yeah, I think it is, `pip install vector-2d` fixed my problem.

**Comment by Conqu3red** [comment]
Hi there - thanks for the report! You are indeed correct, I forgot to add that dependency. Should be fixed in version 1.0.3 of trk-python (https://github.com/Conqu3red/trk-python/commit/400f7e77909167c4b35fbb43d7de20f0e811d751)

This repository has also been updated to pull in the new version of the library (https://github.com/Conqu3red/LR-SVG-Converter/commit/800a3547d7faa2c717836b096700f4b7b25eb3ac)

**Comment by kovasap** [comment]
Thanks!
